packages/features/filters/components/StartTimeFilters.tsx

```diff
@@ -0,0 +1,56 @@
+"use client";
+
+import { usePathname, useRouter } from "next/navigation";
+import { useState } from "react";
+
+import type { Dayjs } from "@calcom/dayjs";
+import dayjs from "@calcom/dayjs";
+import { useFilterQuery } from "@calcom/features/bookings/lib/useFilterQuery";
+import { DateRangePicker } from "@calcom/ui";
+
+export const StartTimeFilters = () => {
+  const router = useRouter();
+  const pathname = usePathname();
+  const { data: query } = useFilterQuery();
+
+  const [afterStartDate, setAfterStartDate] = useState<Dayjs | undefined>(undefined);
+  const [beforeEndDate, setBeforeEndDate] = useState<Dayjs | undefined>(undefined);
+
```

Comment:
we already had `afterStartDate` and `beforeEndDate` in `viewer.bookings.get` query